src/utils/localStorage.js

```diff
@@ -0,0 +1,25 @@
+
+const LOCAL_STORAGE_KEY = 'blogPosts';
+
+export const getPostsFromLocalStorage = () => {
+  const storedPosts = localStorage.getItem(LOCAL_STORAGE_KEY);
+  return storedPosts ? JSON.parse(storedPosts) : [];
+};
+
+export const savePostsToLocalStorage = (posts) => {
+  localStorage.setItem(LOCAL_STORAGE_KEY, JSON.stringify(posts));
+};
```

Comment:
change the name of the function to `setLocalStorage` and make it generic

Reply:
done